modules/ptp-three-card-grandmaster.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/about-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ptp-three-card-grandmaster_{context}"]
+= Using 3 Intel E810 NICs as PTP grandmaster clock
```

Comment:
Because this is a concept, try to reword the title so that it doesn't start with a gerund. One thought I had was "3-card grandmaster clocks".